lib/SILOptimizer/IPO/DeadFunctionElimination.cpp

```diff
@@ -556,6 +556,12 @@ class DeadFunctionAndGlobalElimination {
         auto *fd = getBaseMethod(
             cast<AbstractFunctionDecl>(entry.getMethod().getDecl()));
 
+        // In Embedded Swift, we don't expect SILFunction without definitions on
+        // vtable entries. Having one means the base method was DCE'd already,
+        // so let's avoid marking it alive in the subclass vtable either.
+        bool embedded = Module->getOptions().EmbeddedSwift;
```

Comment:
@eeckstein I wonder if you think this is the right approach? Alternatively, is there a way to have the subclass vtable not contain the (removed in the base class) entry in the first place?